scripts/dev/templates/agent/Dockerfile.ubi

```diff
@@ -3,8 +3,9 @@
 {% set base_image = "registry.access.redhat.com/ubi8/ubi-minimal" %}
 
 {% block packages -%}
-RUN microdnf install -y  --disableplugin=subscription-manager curl \
-    hostname nss_wrapper tar gzip procps\
+RUN microdnf install -y --disableplugin=subscription-manager --setopt=install_weak_deps=0 nss_wrapper
```

Comment:
[Fedora docs](https://docs.fedoraproject.org/en-US/packaging-guidelines/WeakDependencies/) mention setting this option to `False`, which translates to 1. So I'm not entirely sure this options works correctly here.

Would you mind to do a double-verification step and post a diff over the `rpm -qa` before and after this change?

Reply:
Sure, this is `microdnf` though on `ubi-minimal` and I followed the manpages inside the latest container:
```
>> docker run -u root -it --rm --entrypoint="" --name inspect registry.access.redhat.com/ubi8/ubi-minimal:8.9-1029 /bin/sh
sh-4.4# microdnf install --assumeno --disableplugin=subscription-manager --setopt=install_weak_deps=0 nss_wrapper
Downloading metadata...
Downloading metadata...
Downloading metadata...
Package                                                                                                                                                                                         Repository                                                                    Size
Installing:
 cmake-filesystem-3.20.2-5.el8.aarch64                                                                                                                                                          ubi-8-appstream-rpms                                                       45.7 kB
 gdbm-libs-1:1.18-2.el8.aarch64                                                                                                                                                                 ubi-8-baseos-rpms                                                          60.5 kB
 groff-base-1.22.3-18.el8.aarch64                                                                                                                                                               ubi-8-baseos-rpms                                                           1.0 MB
 libpkgconf-1.4.2-1.el8.aarch64                                                                                                                                                                 ubi-8-baseos-rpms                                                          34.5 kB
 ncurses-6.1-10.20180224.el8.aarch64                                                                                                                                                            ubi-8-baseos-rpms                                                         392.5 kB
 nss_wrapper-1.1.13-1.el8.aarch64                                                                                                                                                               ubi-8-appstream-rpms                                                       18.3 kB
 nss_wrapper-libs-1.1.13-1.el8.aarch64                                                                                                                                                          ubi-8-appstream-rpms                                                       40.6 kB
 openssl-1:1.1.1k-9.el8_7.aarch64                                                                                                                                                               ubi-8-baseos-rpms                                                         707.3 kB
 perl-Carp-1.42-396.el8.noarch                                                                                                                                                                  ubi-8-baseos-rpms                                                          30.9 kB
 perl-Data-Dumper-2.167-399.el8.aarch64                                                                                                                                                         ubi-8-baseos-rpms                                                          58.0 kB
 perl-Digest-1.17-395.el8.noarch                                                                                                                                                                ubi-8-appstream-rpms                                                       27.6 kB
 perl-Digest-MD5-2.55-396.el8.aarch64                                                                                                                                                           ubi-8-appstream-rpms                                                       38.1 kB
 perl-Encode-4:2.97-3.el8.aarch64                                                                                                                                                               ubi-8-baseos-rpms                                                           1.5 MB
 perl-Errno-1.28-422.el8.aarch64                                                                                                                                                                ubi-8-baseos-rpms                                                          78.3 kB
 perl-Exporter-5.72-396.el8.noarch                                                                                                                                                              ubi-8-baseos-rpms                                                          34.8 kB
 perl-File-Path-2.15-2.el8.noarch                                                                                                                                                               ubi-8-baseos-rpms                                                          39.0 kB
 perl-File-Temp-0.230.600-1.el8.noarch                                                                                                                                                          ubi-8-baseos-rpms                                                          64.1 kB
 perl-Getopt-Long-1:2.50-4.el8.noarch                                                                                                                                                           ubi-8-baseos-rpms                                                          64.5 kB
 perl-HTTP-Tiny-0.074-2.el8.noarch                                                                                                                                                              ubi-8-baseos-rpms                                                          59.9 kB
 perl-IO-1.38-422.el8.aarch64                                                                                                                                                                   ubi-8-baseos-rpms                                                         145.6 kB
 perl-IO-Socket-IP-0.39-5.el8.noarch                                                                                                                                                            ubi-8-appstream-rpms                                                       48.0 kB
 perl-IO-Socket-SSL-2.066-4.module+el8.3.0+6446+594cad75.noarch                                                                                                                                 ubi-8-appstream-rpms                                                      304.8 kB
 perl-MIME-Base64-3.15-396.el8.aarch64                                                                                                                                                          ubi-8-baseos-rpms                                                          31.4 kB
 perl-Mozilla-CA-20160104-7.module+el8.3.0+6498+9eecfe51.noarch                                                                                                                                 ubi-8-appstream-rpms                                                       15.8 kB
 perl-Net-SSLeay-1.88-2.module+el8.6.0+13392+f0897f98.aarch64                                                                                                                                   ubi-8-appstream-rpms                                                      382.1 kB
 perl-PathTools-3.74-1.el8.aarch64                                                                                                                                                              ubi-8-baseos-rpms                                                          92.1 kB
 perl-Pod-Escapes-1:1.07-395.el8.noarch                                                                                                                                                         ubi-8-baseos-rpms                                                          21.0 kB
 perl-Pod-Perldoc-3.28-396.el8.noarch                                                                                                                                                           ubi-8-baseos-rpms                                                          90.2 kB
 perl-Pod-Simple-1:3.35-395.el8.noarch                                                                                                                                                          ubi-8-baseos-rpms                                                         218.3 kB
 perl-Pod-Usage-4:1.69-395.el8.noarch                                                                                                                                                           ubi-8-baseos-rpms                                                          35.3 kB
 perl-Scalar-List-Utils-3:1.49-2.el8.aarch64                                                                                                                                                    ubi-8-baseos-rpms                                                          68.2 kB
 perl-Socket-4:2.027-3.el8.aarch64                                                                                                                                                              ubi-8-baseos-rpms                                                          60.1 kB
 perl-Storable-1:3.11-3.el8.aarch64                                                                                                                                                             ubi-8-baseos-rpms                                                          96.9 kB
 perl-Term-ANSIColor-4.06-396.el8.noarch                                                                                                                                                        ubi-8-baseos-rpms                                                          47.1 kB
 perl-Term-Cap-1.17-395.el8.noarch                                                                                                                                                              ubi-8-baseos-rpms                                                          23.4 kB
 perl-Text-ParseWords-3.30-395.el8.noarch                                                                                                                                                       ubi-8-baseos-rpms                                                          18.4 kB
 perl-Text-Tabs+Wrap-2013.0523-395.el8.noarch                                                                                                                                                   ubi-8-baseos-rpms                                                          24.7 kB
 perl-Time-Local-1:1.280-1.el8.noarch                                                                                                                                                           ubi-8-baseos-rpms                                                          34.3 kB
 perl-URI-1.73-3.el8.noarch                                                                                                                                                                     ubi-8-appstream-rpms                                                      118.9 kB
 perl-Unicode-Normalize-1.25-396.el8.aarch64                                                                                                                                                    ubi-8-baseos-rpms                                                          80.3 kB
 perl-constant-1.33-396.el8.noarch                                                                                                                                                              ubi-8-baseos-rpms                                                          26.0 kB
 perl-interpreter-4:5.26.3-422.el8.aarch64                                                                                                                                                      ubi-8-baseos-rpms                                                           6.6 MB
 perl-libnet-3.11-3.el8.noarch                                                                                                                                                                  ubi-8-appstream-rpms                                                      123.8 kB
 perl-libs-4:5.26.3-422.el8.aarch64                                                                                                                                                             ubi-8-baseos-rpms                                                           1.5 MB
 perl-macros-4:5.26.3-422.el8.aarch64                                                                                                                                                           ubi-8-baseos-rpms                                                          74.3 kB
 perl-parent-1:0.237-1.el8.noarch                                                                                                                                                               ubi-8-baseos-rpms                                                          20.5 kB
 perl-podlators-4.11-1.el8.noarch                                                                                                                                                               ubi-8-baseos-rpms                                                         120.8 kB
 perl-threads-1:2.21-2.el8.aarch64                                                                                                                                                              ubi-8-baseos-rpms                                                          61.5 kB
 perl-threads-shared-1.58-2.el8.aarch64                                                                                                                                                         ubi-8-baseos-rpms                                                          48.1 kB
 pkgconf-1.4.2-1.el8.aarch64                                                                                                                                                                    ubi-8-baseos-rpms                                                          38.2 kB
 pkgconf-m4-1.4.2-1.el8.noarch                                                                                                                                                                  ubi-8-baseos-rpms                                                          17.5 kB
 pkgconf-pkg-config-1.4.2-1.el8.aarch64                                                                                                                                                         ubi-8-baseos-rpms                                                          15.6 kB
Transaction Summary:
 Installing:       52 packages
 Reinstalling:      0 packages
 Upgrading:         0 packages
 Obsoleting:        0 packages
 Removing:          0 packages
 Downgrading:       0 packages
Enabling module streams:
    perl-IO-Socket-SSL:2.066
    perl-libwww-perl:6.34


Operation aborted.
sh-4.4# microdnf install --assumeno --disableplugin=subscription-manager nss_wrapper
Package                                                                                                                                                                                         Repository                                                                    Size
Installing:
 cmake-3.20.2-5.el8.aarch64                                                                                                                                                                     ubi-8-appstream-rpms                                                       10.8 MB
 cmake-data-3.20.2-5.el8.noarch                                                                                                                                                                 ubi-8-appstream-rpms                                                        1.8 MB
 cmake-filesystem-3.20.2-5.el8.aarch64                                                                                                                                                          ubi-8-appstream-rpms                                                       45.7 kB
 cmake-rpm-macros-3.20.2-5.el8.noarch                                                                                                                                                           ubi-8-appstream-rpms                                                       45.0 kB
 emacs-filesystem-1:26.1-11.el8.noarch                                                                                                                                                          ubi-8-baseos-rpms                                                          71.8 kB
 expat-2.2.5-11.el8.aarch64                                                                                                                                                                     ubi-8-baseos-rpms                                                         106.2 kB
 gdbm-1:1.18-2.el8.aarch64                                                                                                                                                                      ubi-8-baseos-rpms                                                         130.7 kB
 gdbm-libs-1:1.18-2.el8.aarch64                                                                                                                                                                 ubi-8-baseos-rpms                                                          60.5 kB
 groff-base-1.22.3-18.el8.aarch64                                                                                                                                                               ubi-8-baseos-rpms                                                           1.0 MB
 libnsl2-1.2.0-2.20180605git4a062cf.el8.aarch64                                                                                                                                                 ubi-8-baseos-rpms                                                          56.5 kB
 libpkgconf-1.4.2-1.el8.aarch64                                                                                                                                                                 ubi-8-baseos-rpms                                                          34.5 kB
 libtirpc-1.1.4-8.el8.aarch64                                                                                                                                                                   ubi-8-baseos-rpms                                                         111.9 kB
 libuv-1:1.41.1-1.el8_4.aarch64                                                                                                                                                                 ubi-8-appstream-rpms                                                      153.4 kB
 make-1:4.2.1-11.el8.aarch64                                                                                                                                                                    ubi-8-baseos-rpms                                                         502.1 kB
 ncurses-6.1-10.20180224.el8.aarch64                                                                                                                                                            ubi-8-baseos-rpms                                                         392.5 kB
 nss_wrapper-1.1.13-1.el8.aarch64                                                                                                                                                               ubi-8-appstream-rpms                                                       18.3 kB
 nss_wrapper-libs-1.1.13-1.el8.aarch64                                                                                                                                                          ubi-8-appstream-rpms                                                       40.6 kB
 openssl-1:1.1.1k-9.el8_7.aarch64                                                                                                                                                               ubi-8-baseos-rpms                                                         707.3 kB
 perl-Carp-1.42-396.el8.noarch                                                                                                                                                                  ubi-8-baseos-rpms                                                          30.9 kB
 perl-Data-Dumper-2.167-399.el8.aarch64                                                                                                                                                         ubi-8-baseos-rpms                                                          58.0 kB
 perl-Digest-1.17-395.el8.noarch                                                                                                                                                                ubi-8-appstream-rpms                                                       27.6 kB
 perl-Digest-MD5-2.55-396.el8.aarch64                                                                                                                                                           ubi-8-appstream-rpms                                                       38.1 kB
 perl-Encode-4:2.97-3.el8.aarch64                                                                                                                                                               ubi-8-baseos-rpms                                                           1.5 MB
 perl-Errno-1.28-422.el8.aarch64                                                                                                                                                                ubi-8-baseos-rpms                                                          78.3 kB
 perl-Exporter-5.72-396.el8.noarch                                                                                                                                                              ubi-8-baseos-rpms                                                          34.8 kB
 perl-File-Path-2.15-2.el8.noarch                                                                                                                                                               ubi-8-baseos-rpms                                                          39.0 kB
 perl-File-Temp-0.230.600-1.el8.noarch                                                                                                                                                          ubi-8-baseos-rpms                                                          64.1 kB
 perl-Getopt-Long-1:2.50-4.el8.noarch                                                                                                                                                           ubi-8-baseos-rpms                                                          64.5 kB
 perl-HTTP-Tiny-0.074-2.el8.noarch                                                                                                                                                              ubi-8-baseos-rpms                                                          59.9 kB
 perl-IO-1.38-422.el8.aarch64                                                                                                                                                                   ubi-8-baseos-rpms                                                         145.6 kB
 perl-IO-Socket-IP-0.39-5.el8.noarch                                                                                                                                                            ubi-8-appstream-rpms                                                       48.0 kB
 perl-IO-Socket-SSL-2.066-4.module+el8.3.0+6446+594cad75.noarch                                                                                                                                 ubi-8-appstream-rpms                                                      304.8 kB
 perl-MIME-Base64-3.15-396.el8.aarch64                                                                                                                                                          ubi-8-baseos-rpms                                                          31.4 kB
 perl-Mozilla-CA-20160104-7.module+el8.3.0+6498+9eecfe51.noarch                                                                                                                                 ubi-8-appstream-rpms                                                       15.8 kB
 perl-Net-SSLeay-1.88-2.module+el8.6.0+13392+f0897f98.aarch64                                                                                                                                   ubi-8-appstream-rpms                                                      382.1 kB
 perl-PathTools-3.74-1.el8.aarch64                                                                                                                                                              ubi-8-baseos-rpms                                                          92.1 kB
 perl-Pod-Escapes-1:1.07-395.el8.noarch                                                                                                                                                         ubi-8-baseos-rpms                                                          21.0 kB
 perl-Pod-Perldoc-3.28-396.el8.noarch                                                                                                                                                           ubi-8-baseos-rpms                                                          90.2 kB
 perl-Pod-Simple-1:3.35-395.el8.noarch                                                                                                                                                          ubi-8-baseos-rpms                                                         218.3 kB
 perl-Pod-Usage-4:1.69-395.el8.noarch                                                                                                                                                           ubi-8-baseos-rpms                                                          35.3 kB
 perl-Scalar-List-Utils-3:1.49-2.el8.aarch64                                                                                                                                                    ubi-8-baseos-rpms                                                          68.2 kB
 perl-Socket-4:2.027-3.el8.aarch64                                                                                                                                                              ubi-8-baseos-rpms                                                          60.1 kB
 perl-Storable-1:3.11-3.el8.aarch64                                                                                                                                                             ubi-8-baseos-rpms                                                          96.9 kB
 perl-Term-ANSIColor-4.06-396.el8.noarch                                                                                                                                                        ubi-8-baseos-rpms                                                          47.1 kB
 perl-Term-Cap-1.17-395.el8.noarch                                                                                                                                                              ubi-8-baseos-rpms                                                          23.4 kB
 perl-Text-ParseWords-3.30-395.el8.noarch                                                                                                                                                       ubi-8-baseos-rpms                                                          18.4 kB
 perl-Text-Tabs+Wrap-2013.0523-395.el8.noarch                                                                                                                                                   ubi-8-baseos-rpms                                                          24.7 kB
 perl-Time-Local-1:1.280-1.el8.noarch                                                                                                                                                           ubi-8-baseos-rpms                                                          34.3 kB
 perl-URI-1.73-3.el8.noarch                                                                                                                                                                     ubi-8-appstream-rpms                                                      118.9 kB
 perl-Unicode-Normalize-1.25-396.el8.aarch64                                                                                                                                                    ubi-8-baseos-rpms                                                          80.3 kB
 perl-constant-1.33-396.el8.noarch                                                                                                                                                              ubi-8-baseos-rpms                                                          26.0 kB
 perl-interpreter-4:5.26.3-422.el8.aarch64                                                                                                                                                      ubi-8-baseos-rpms                                                           6.6 MB
 perl-libnet-3.11-3.el8.noarch                                                                                                                                                                  ubi-8-appstream-rpms                                                      123.8 kB
 perl-libs-4:5.26.3-422.el8.aarch64                                                                                                                                                             ubi-8-baseos-rpms                                                           1.5 MB
 perl-macros-4:5.26.3-422.el8.aarch64                                                                                                                                                           ubi-8-baseos-rpms                                                          74.3 kB
 perl-parent-1:0.237-1.el8.noarch                                                                                                                                                               ubi-8-baseos-rpms                                                          20.5 kB
 perl-podlators-4.11-1.el8.noarch                                                                                                                                                               ubi-8-baseos-rpms                                                         120.8 kB
 perl-threads-1:2.21-2.el8.aarch64                                                                                                                                                              ubi-8-baseos-rpms                                                          61.5 kB
 perl-threads-shared-1.58-2.el8.aarch64                                                                                                                                                         ubi-8-baseos-rpms                                                          48.1 kB
 pkgconf-1.4.2-1.el8.aarch64                                                                                                                                                                    ubi-8-baseos-rpms                                                          38.2 kB
 pkgconf-m4-1.4.2-1.el8.noarch                                                                                                                                                                  ubi-8-baseos-rpms                                                          17.5 kB
 pkgconf-pkg-config-1.4.2-1.el8.aarch64                                                                                                                                                         ubi-8-baseos-rpms                                                          15.6 kB
 platform-python-3.6.8-56.el8_9.aarch64                                                                                                                                                         ubi-8-baseos-rpms                                                          89.1 kB
 platform-python-pip-9.0.3-23.el8.noarch                                                                                                                                                        ubi-8-baseos-rpms                                                           1.6 MB
 platform-python-setuptools-39.2.0-7.el8.noarch                                                                                                                                                 ubi-8-baseos-rpms                                                         647.4 kB
 python3-libs-3.6.8-56.el8_9.aarch64                                                                                                                                                            ubi-8-baseos-rpms                                                           8.1 MB
 python3-pip-wheel-9.0.3-23.el8.noarch                                                                                                                                                          ubi-8-baseos-rpms                                                         886.8 kB
 python3-setuptools-wheel-39.2.0-7.el8.noarch                                                                                                                                                   ubi-8-baseos-rpms                                                         296.0 kB
 vim-filesystem-2:8.0.1763-19.el8_6.4.noarch                                                                                                                                                    ubi-8-appstream-rpms                                                       51.7 kB
Transaction Summary:
 Installing:       69 packages
 Reinstalling:      0 packages
 Upgrading:         0 packages
 Obsoleting:        0 packages
 Removing:          0 packages
 Downgrading:       0 packages
Enabling module streams:
    perl-IO-Socket-SSL:2.066
    perl-libwww-perl:6.34


Operation aborted.
sh-4.4#
```

Reply:
Ok then! This must have been some misalignment in Red Hat docs. LGTM!